pawsitive/db/schema.rb

```diff
@@ -10,11 +10,24 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_05_15_130200) do
+ActiveRecord::Schema[7.1].define(version: 2024_05_22_015823) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  create_table "accounts", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "user_id", null: false
+    t.string "address"
+    t.string "country"
+    t.string "city"
+    t.string "state"
+    t.string "zip_or_postcode"
+    t.date "birthday"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_accounts_on_user_id"
+  end
```

Comment:
Hi @Gladdy17, now that we add the extra columns to the users table this accounts table should be remove.

Reply:
this one i will remove in the current branch i am working on